File: SchoolApiService/ViewModels/CreateExamScheduleStandardVM.cs
namespace SchoolApiService.ViewModels
{
    public class CreateExamScheduleStandardVM
    {
        public int ExamScheduleId { get; set; }
        public int StandardId { get; set; }
        public IEnumerable<CreateExamSubjectWithScheduleVM>? ExamSubjects { get; set; }
    }
}

File: SchoolAppClient.NG/src/app/Models/create-exam-schedule-standard-vm.ts
import { CreateExamSubjectWithScheduleVM } from "./create-exam-subject-with-schedule-vm";

export class CreateExamScheduleStandardVM {
 
  examScheduleId!: number;
  standardId!: number;
  examSubjects: CreateExamSubjectWithScheduleVM[] = [];
}
